src/components/footer/_footer.scss

```diff
@@ -31,22 +31,24 @@
   }
 
   .govuk-footer__link {
-    @include govuk-focusable-fill;
+    @include govuk-focusable-text-link;
 
     &:link,
     &:visited {
       color: $govuk-footer-link;
+      text-decoration: none;
```

Comment:
@dashouse are we sure about removing the underline in the footer, it makes all the links look like body content, there's no way of knowing something is clickable.

Reply:
hmm im presuming this change it to make the footer consistent with the header...

Reply:
Decided to go with this for now since it fits our navigation patterns.

I believe that this could fail WCAG though:

https://www.w3.org/TR/WCAG20-TECHS/F73.html